cmake/common.cmake

```diff
@@ -748,3 +748,49 @@ function(add_picobench name)
 
   set_property(TEST ${name} PROPERTY LABELS benchmark)
 endfunction()
+
+# Perf tool executable
+set(SUBMITTER_DIR ${CCF_DIR}/tests/perf-system/submitter)
+
+add_executable(
+  submit ${SUBMITTER_DIR}/submit.cpp ${SUBMITTER_DIR}/handle_arguments.hpp
+         ${SUBMITTER_DIR}/parquet_data.hpp
+)
+
+add_library(stdcxxhttp_parser.host "${HTTP_PARSER_SOURCES}")
```

Comment:
I added a similar comment on the ansible PR, but I think it got lost:

Do we still need stdcxx? Do we still need to rebuild this and attempt to link with a different std lib implementation? If we're building within the CCF codebase, we should be able to use the same compiler toolchain as the other host libraries.

Reply:
Sorry for not addressing your comment on the previous PR.
So, the problem is that libarrow-dev and libparquet-dev that contain the header files for arrow and parquet are compiled against libstdcpp. In the beginning, when trying to compile my program, depending on arrow and parquet, with libcpp I got errors coming form libarrow. Discussing the compiling issues with @achamayou, he thought that we should best avoid rebuilding arrow using libcpp and instead, since there are no security issues for the perf-tool, build crypto and http_parser against libstdcpp.
@achamayou would probably be able to explain the issue even better in case I missed or misunderstood something.

Reply:
Thanks for the summary, @achamayou explained the same. I had completely missed this dependency, but it makes sense, so the std library variants stay!